internal/resource/resource.go

```diff
@@ -87,6 +170,12 @@ func NewResource(ctx context.Context, renv *readiness.Env, slice *apiv1.Resource
 	return res, nil
 }
 
+type patchMeta struct {
+	APIVersion string          `json:"apiVersion"`
+	Kind       string          `json:"kind"`
+	Ops        jsonpatch.Patch `json:"ops"`
```

Comment:
Slice of patches?

Reply:
jsonpatch.Patch is (surprisingly) already a slice of patch operations.